### PR TITLE
feat(config): Optimize main_net_config.conf

### PR DESCRIPTION
**Pull request by forfreeday**
**What does this PR do?**

1. Remove Failed Configurations
2. Add the latest valid configurations

**Why are these changes required?**
Optimize the configuration so that developers can use the configuration properly

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: main_net_config.conf [hunk 1]
```diff
@@ -464,20 +578,30 @@ block = {
   proposalExpireTime = 259200000 // 3 day: 259200000(ms)
 }
 
-# Transaction reference block, default is "head", configure to "solid" can avoid TaPos error
-trx.reference.block = "solid" // head;solid;
+# Transaction reference block, default is "solid", configure to "head" may accur TaPos error
```
Comment (lurais) [inline main_net_config.conf:581]: Is accur anything else?

File: main_net_config.conf [hunk 2]
```diff
@@ -127,19 +159,18 @@ node {
...
+
   maxConnectionsWithSameIp = 2
 
   maxHttpConnectNumber = 50
 
   minParticipationRate = 15
 
-  receiveTcpMinDataLength = 2048
-  isOpenFullTcpDisconnect = true
+  isOpenFullTcpDisconnect = false
```
Comment (halibobo1205) [inline main_net_config.conf:173]: why  is false ?

File: main_net_config.conf [hunk 3]
```diff
@@ -464,20 +578,30 @@ block = {
...
+  maxEnergyLimitForConstant = 100000000
   minTimeRatio = 0.0
   maxTimeRatio = 5.0
   saveInternalTx = false
 
+  # Indicates whether the node stores featured internal transactions, such as freeze, vote and so on
+  # saveFeaturedInternalTx = false
+
   # In rare cases, transactions that will be within the specified maximum execution time (default 10(ms)) are re-executed and packaged
   # longRunningTime = 10
```
Comment (halibobo1205) [inline main_net_config.conf:598]: `longRunningTime` for what ?

File: main_net_config.conf (hunk 2 above)
Comment (forfreeday) [reply main_net_config.conf:173]: This function is to randomly disconnect the TCP connection, the current verification function is not very good, so it is set to `false`, and we will consider removing this function in the future.

File: main_net_config.conf (hunk 1 above)
Comment (forfreeday) [reply main_net_config.conf:581]: This is a cheap mistake, it should read: incur

File: main_net_config.conf (hunk 3 above)
Comment (forfreeday) [reply main_net_config.conf:598]: The purpose of this configuration is to specify the maximum execution time for a contract transaction, as described in .
This configuration is being used temporarily due to a feature adjustment.

**Pull request merged**